assets/performanceprofile/scripts/cpuset-configure.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# cpuset-configure.sh configures three cpusets in preparation for allowing containers to have cpu load balancing disabled.
+# To configure a cpuset to have load balance disabled (on cgroup v1), a cpuset cgroup must have `cpuset.sched_load_balance`
+# set to 0 (disable), and any cpuset that contains the same set as `cpuset.cpus` must also have `cpuset.sched_load_balance` set to disabled.
+
+set -euo pipefail
+
+root=/sys/fs/cgroup/cpuset
+system="$root"/system
```

Comment:
I think the commonly used name is "system.slice".

Reply:
I specifically chose not to use `.slice` because slices are something unique to systemd and the name would imply it's being managed by systemd, which it's not

Reply:
I specifically chose not to use `.slice` because slices are something unique to systemd and the name would imply it's being managed by systemd, which it's not